C#/Domain/Models/Patient.cs

```diff
@@ -4,10 +4,13 @@ public class Patient
 {
     public string Name { get; set; }
 
+    public int Id;
```

Comment:
Could u put the {get;set;} and don't you need the [Key] for database? I can see that there is only one key in SensorValue, but not anywhere else and I am a bit confused since the last time I saw the code on disc there was this attribute to them.

Reply:
In the HospitalContext class, we designate the primary keys by the HasKey method. So this is not necessary, I will remove the attribute [Key].

I am unsure about the foreign keys if they are automatically generated from the relationship by having a List of values fx.